_dev/src/ts/pages/Home.ts

```diff
@@ -0,0 +1,56 @@
+import PageAbstract from './PageAbstract';
+
+export default class HomePage extends PageAbstract {
+  form: HTMLFormElement | null;
+  submitButton: HTMLButtonElement | null;
+
+  constructor() {
+    super();
+    this.form = document.forms.namedItem('next_page');
+    this.submitButton = this.getSubmitButton();
+  }
+
+  public mount = () => {
+    if (this.form) {
```

Comment:
If the form is not found, can we consider this as normal? Or shall we trigger an exception?